lib/renderer/bridge/session.go

```diff
@@ -33,19 +48,15 @@ func (s *session) Identifier() string {
 }
 
 func (s *session) sendMessage(msg string) error {
-	s.lock.Lock()
-	defer s.lock.Unlock()
-
-	if err := s.conn.WriteMessage(websocket.TextMessage, []byte(msg)); err != nil {
-		s.log.Debug(err.Error())
-		return err
+	if !s.done {
+		s.writeChan <- msg
```

Comment:
Let's convert it here as this only happens once. A zero copy way to do this (I think) is to do:
`*(*[]byte)(unsafe.Pointer(&msg))`. We can only do this because this string is read only.